libs/schemas/src/entities/group.schemas.ts

```diff
@@ -7,9 +7,10 @@ const ContractSource = z.object({
     BalanceSourceType.ERC20,
     BalanceSourceType.ERC721,
     BalanceSourceType.ERC1155,
+    BalanceSourceType.SPL,
   ]),
   evm_chain_id: PG_INT,
-  contract_address: z.string().regex(/^0x[a-fA-F0-9]{40}$/),
+  contract_address: z.string().regex(/^[a-zA-Z0-9]{32,44}$/),
```

Comment:
This really loosens the schema for EVM addresses which may be a problem if we don't validate the contract address elsewhere in the API. Can you check if we validate EVM addresses in the API before creating a group in the DB - can resolve if we do.

Reply:
Updated the requriementSchema_V1 in the last commit to conditionally filter and confirmed it works correctly on evm vs solana communities. The schema in this file seems to have no effect on validation of requirements in general.